src/plugins/minimal_scene/MinimalScene.hh

```diff
@@ -368,6 +371,12 @@ namespace gz::gui::plugins
     /// \param[in] _sky True to enable the sky, false otherwise.
     public: void SetSkyEnabled(const bool &_sky);
 
+    /// @brief  \brief Set the shadow texture size for the given light type.
+    /// @param _lightType Light type that creates the shadow
+    /// @param _textureSize Shadow texture size
```

Comment:
nit: replace `@` with `\` for consistency

Reply:
0bc87f8c8ff8b2023472279cad3d85bf4b447701